Automattic/jetpack
# Update plugins menu destinations to calypso pages

Fixes https://github.com/Automattic/wp-calypso/issues/59734

#### Changes proposed in this Pull Request:

* Change the destination of the manage plugin page to the calypso screen plugin screen. This is mainly WPCOM related changes.

#### Jetpack product discussion



#### Does this pull request to change what data or activity we track or use?



#### Testing instructions:






- in your atomic dev blog, copy the changes proposed in this PR 
- visit your site

Free Atomic: 
- make sure that `Plugins` links to Calypso https://wordpress.com/plugins/[DOMAIN]
- You don't see any submenu under `Plugins` menu

Atomic with a higher plan:
- make sure that `Plugins -> Add New` still links to Calypso https://wordpress.com/plugins/[DOMAIN]
- make sure that `Plugins` links to the default WP admin `wp-admin/plugins.php` page.
- make sure that `Plugins -> Installed Plugins` to the default WP admin `wp-admin/plugins.php` page.
